oops/Second.rb

```diff
@@ -0,0 +1,16 @@
+# Write a program to access a global variable from a method of the class.
+
+# defined a global var
+$global = 2024
+
+class Second
+
+  # define a method to return global var
+  def access_global()
```

Comment:
@sabyasachi-palit , please later check the file names, class, and variables names if it is following best practices